.github/workflows/image.yaml

```diff
@@ -4,6 +4,10 @@ on:
   push:
     branches:
       - master
+  pull_request:
+    branches:
+      - master
+    types: [ labeled, unlabeled, opened, synchronize, reopened ]
```

Comment:
Adding `labeled` and `unlabeled` types to catch adding and removing `test-arm-image` label.